src/model/bioprotocol.v1.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: BioProtocol
+type: array
+items:
+    type: object
+    properties:
+        protocol_sequencing_number:
+            description: "section ID, looks like 's4-1'"
+            type: string
+        protocol_title:
```

Comment:
is the `protocol_` prefix in these two fields necessary? Thinking about e.g. [podcast episodes](https://github.com/elifesciences/api-raml/blob/develop/src/snippets/podcast-episode.v1.yaml#L5) where we use specific terms like `number` without a prefix.

Reply:
I can remove those